2_Operations/work-process.md

```diff
@@ -62,14 +62,17 @@ As an attendee, ask yourself:
 - Did I find a handy tool/method/module that I think we should use?
 - Do I have an idea about implementation we will have in the next sprint, and I want to discuss it?
 - Have I seen a great technical talk and I want to share it so we can discuss it the next time?
+- Do I want to share something I leared in a short lighning talk?
+- Do I need help with the user story I have been asigned to?
 
 As a curator:
 
-- Announce in #operations Zoom session to join into
+- Announce in #development Zoom session to join into
 - As a curator, meet and greet everyone, especially first-time joiners. 
+- Annonce who is having Lightning talk if one was prepared.
 - If you hear crickets, start rolling the ball with a question, an anecdote or a funny story.
 - Make sure no one is talking the whole 30 minutes, everyone should have an opinion!
-- After 30 minutes or less announce that there is time for Lightning talks.
+- Write a report into Recurring ticket for the meeting.
```

Comment:
I'd write "into _the_ recurring ticket" even if Grammarly doesn't raise it here.